### PR TITLE
Add support to css modules

### PR DESCRIPTION
**Pull request by ArthurSampaio**
#### What is the purpose of this pull request?

Remove global handlers in order to use CSS modules.

#### How should this be manually tested?
https://carouselcssmodules--storecomponents.myvtex.com/

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/Carousel.tsx [hunk 1]
```diff
@@ -207,7 +208,7 @@ export default class Carousel extends Component<Props> {
     }
 
     return (
-      <div className="vtex-carousel force-full-width">
+      <div className={`${carousel.container} force-full-width`}>
```
Comment (brunojdo) [inline react/Carousel.tsx]: this tag was removed `force-full-width`

File: react/global.css [hunk 2]
```diff
@@ -1,65 +1,65 @@
-/* General */
-.vtex-carousel{
+/* General */	
+.vtex-carousel{
```
Comment (brunojdo) [inline react/global.css]: if is on CSS modules then remove from global

File: react/Carousel.tsx (hunk 1 above)
Comment (GustavoMA) [reply react/Carousel.tsx]: Fixed in the rebase.

File: react/global.css (hunk 2 above)
Comment (GustavoMA) [reply react/global.css]: Done.

**Pull request closed without merging**